core/src/main/scala/better/files/Implicits.scala

```diff
@@ -115,14 +115,17 @@ trait Implicits extends ManagedResource.FlatMap.Implicits with Scanner.Read.Impl
         bufferSize: Int = DefaultBufferSize
       ): ObjectInputStream =
       new ObjectInputStream(if (bufferSize <= 0) in else buffered(bufferSize)) {
-        override protected def resolveClass(objectStreamClass: ObjectStreamClass): Class[_] =
+        override protected def resolveClass(_objectStreamClass: ObjectStreamClass | Null): Class[_] = {
+          val objectStreamClass = _objectStreamClass.nn
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.